core/src/test/java/com/datastax/oss/driver/internal/core/loadbalancing/DefaultLoadBalancingPolicyInitTest.java

```diff
@@ -129,6 +130,9 @@ public void should_warn_if_contact_points_not_in_local_dc() {
   public void should_include_nodes_from_local_dc() {
     // Given
     when(metadataManager.getContactPoints()).thenReturn(ImmutableSet.of(node1, node2));
+    when(node1.getState()).thenReturn(NodeState.UP);
```

Comment:
maybe we can name `node1` - `upNode`, `node2` -> `downNode`, `node3` -> `uknownNode` for better readability

Reply:
These are instance fields and they are used in other tests as well, so I guess it would be difficult to rename them since they might play different roles in different tests.